src/FakeItEasy/Core/TypeCatalogue.cs

```diff
@@ -1,4 +1,4 @@
-namespace FakeItEasy.Core
+﻿namespace FakeItEasy.Core

```

Comment:
I see that VS2017 doesn't respect the lack of BOM (as specified by .editorconfig).
We should probably just not worry about it, no?